server/controllers/guide.js

```diff
@@ -49,7 +49,7 @@ var readUserGuides = function (req, res, next) {
   Guide.findAll({
     where: {
       // need to have front end send userId from localStorage
-      userId: req.userId
+      userId: req.headers.userid
```

Comment:
Based on the property in the header, looks like this should be .userId (capital I)